File: phase01/immortals_repo/harness/scenarioconductor/data/adaptationresult.py
# noinspection PyPep8Naming
class AdaptationResult:
    """
    :type adaptationStatusValue: str
    :type audits: list[str]
    :type auditsAsString: str
    :type details: str
    :type selectedDfu: str
    :type sessionIdentifier: str
    """

    @classmethod
    def from_dict(cls, d):
        return cls(**d)

    def __init__(self,
                 adaptationStatusValue,
                 audits,
                 auditsAsString,
                 details,
                 selectedDfu,
                 sessionIdentifier
                 ):
        self.adaptationStatusValue = adaptationStatusValue
        self.audits = audits
        self.auditsAsString = auditsAsString
        self.details = details
        self.selectedDfu = selectedDfu
        self.sessionIdentifier = sessionIdentifier

    def to_dict(self):
        return {
            'adaptationStatusValue': self.adaptationStatusValue,
            'audits': self.audits,
            'auditsAsString': self.auditsAsString,
            'details': self.details,
            'selectedDfu': self.selectedDfu,
            'sessionIdentifier': self.sessionIdentifier
        }

File: immortals_repo/applications/server/Marti/validation/mil/darpa/immortals/core/api/ll/phase1/AdaptationResult.java
package mil.darpa.immortals.core.api.ll.phase1;

import java.util.LinkedList;

/**
 * Created by awellman@bbn.com on 9/11/17.
 */
public class AdaptationResult {
    public final String adaptationStatusValue;
    public final LinkedList<String> audits;
    public final String auditsAsString;
    public final String details;
    public final String selectedDfu;
    public final String sessionIdentifier;

    public AdaptationResult(String adaptationStatusValue, LinkedList<String> audits, String auditsAsString,
                            String details, String selectedDfu, String sessionIdentifier) {
        this.adaptationStatusValue = adaptationStatusValue;
        this.audits = audits;
        this.auditsAsString = auditsAsString;
        this.details = details;
        this.selectedDfu = selectedDfu;
        this.sessionIdentifier = sessionIdentifier;
    }
}
